pep-0678.rst

```diff
@@ -165,8 +165,9 @@ subject to breakage without warning
 <https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers>`__.
 We are also unaware of any code which *would* be broken by adding ``__notes__``.
 
-While ``add_note()`` is a fairly common method name, our Google and GitHub
-searches did not find any exception classes which define such a method.
+While ``add_note()`` is a fairly common method name, our Google and `GitHub
```

Comment:
I can understand why it may be confusing, but what the text seems to be saying is that it is a common method name _overall_, it didn't appear to be used for any _execption subclasses_ in particular.

Reply:
Yep.  Might be clearer to say:

> We were also unable to find any code which would be broken by the addition of `BaseException.add_note()` - while searching Google and GitHub finds several definitions of an `.add_note()` method, none of them are on a subclass of `BaseException`.